ipi/inputs/normalmodes.py

```diff
@@ -87,8 +87,8 @@ class InputNormalModes(Input):
                                    "options": ['fft', 'matrix']}),
         "propagator": (InputAttribute, {"dtype": str,
                                    "default": "exact",
-                                   "help": "How to propagate the free ring polymer dynamics. Cayley transform is not exact but is strongly stable and avoid potential resonance issues.",
-                                   "options": ['exact', 'cayley']})
+                                   "help": "How to propagate the free ring polymer dynamics. Cayley transform is not exact but is strongly stable and avoid potential resonance issues. A bab scheme is also added to perform numerical verlet type propagation.",
```

Comment:
Maybe add here that only bab works for bosons? (is that right?)

Reply:
Actually no, it works also for distinguishable particles, but is the only one that works for bosons.

Reply:
Ok, then can you be more verbose in this respect in the places that this is called? Especially in the documentation?  I tried to submit a comment there too but it is not showing up here.

Reply:
OK, no problem. Please see the changes I just made below. Is that better? I think that these should be automatically included in the documentation but I'm not sure. If you want me to add more info anywhere else, could you point to the right place?